components/SpotDetails.js

```diff
@@ -129,79 +142,108 @@ export default function SpotDetails({ spot, addSpot, deleteSpot, editSpot }) {
           editSpot={editSpot}
         />
       )}
-    </DetailsSection>
+    </SpotDetailsSection>
   );
 }
 
-const DetailsSection = styled.section`
+/* ↓↓ Styling ↓↓ */
+
+const SpotDetailsSection = styled.section`
   width: 100%;
   padding: 10px;
   font-size: 14px;
 `;
 
+const DetailsHeadline = styled.h2`
+  text-align: center;
+  color: var(--third-color);
+  font-size: 18px;
+  font-weight: 500;
+  text-shadow: 0.5px 0.5px 1px lightgray;
+`;
+
+const DetailTitle = styled.h3`
+  color: var(--secondary-color);
+`;
+
+const AddresseIcon = styled.svg`
+  margin-right: 5px;
+`;
+
+const AddresseLink = styled(Link)`
+  color: var(--first-font-color);
+  font-size: 16px;
+  text-decoration: none;
+
+  :hover {
+    cursor: pointer;
+    filter: drop-shadow(1px 1px 1px rgb(0 0 0 / 0.1));
+  }
+`;
+
 const DetailsSubList = styled.ul`
   font-size: 16px;
   padding: 5px;
   list-style: none;
   padding: 0 10px 0 10px;
+  color: var(--first-font-color);
 `;
 
 const DetailsParagraph = styled.p`
   text-align: justify;
   padding: 0 10px 0 10px;
```

Comment:
You could use the shorthand here:
```suggestion
  padding: 0 10px;
```